staging/src/k8s.io/apimachinery/pkg/watch/mux.go

```diff
@@ -225,6 +227,7 @@ func (m *Broadcaster) distribute(event Event) {
 			case w.result <- event:
 			case <-w.stopped:
 			default: // Don't block if the event can't be queued.
+				glog.Warningf("drop event: %v", event)
```

Comment:
nit: should it be drop**ping** event?

Reply:
@nikhita maybe only drop is enough? or dropped?

Reply:
no strong opinions. dropped sounds better to me though.

Reply:
I prefer to have `dropping` here.